routes/web.php

```diff
@@ -15,6 +15,10 @@
     return view('mainpage');
 });
 
+Route::get('/info', function() {
+    return view('info');
```

Comment:
Give route a name,` ->name('info') ` might be good, so we can more easily reference it in code later.